src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -9,61 +9,27 @@
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
-import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 
 @Entity
-@Table(name = "movie_sessions")
+@Data
+@NoArgsConstructor
+@RequiredArgsConstructor
 public class MovieSession {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @ManyToOne(fetch = FetchType.LAZY)
+    @NonNull
     private Movie movie;
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "cinema_hall_id")
+    @JoinColumn(name = "cinemaHall_id")
```

Comment:
it's better not to change this naming. snake case (cinema_hall_id) in this situation is better for SQL row than cinemaHall_id